monitor/src/main/java/edu/asu/diging/monitor/core/service/impl/NotificationManager.java

```diff
@@ -51,6 +53,14 @@ public boolean addRecipient(String name, String email) throws EmailAlreadyRegist
 		return true;
 	}
 	
+	@Override
+	public Map<String,String> showRecipients(){
+		INotificationRecipient[] recipients = dbConnection.getAllRecipients();
+		Map<String,String> recipientInfo = new HashMap<>();
+		Arrays.asList(recipients).forEach(x -> recipientInfo.put(x.getName(),x.getEmail()));
+		return recipientInfo;
```

Comment:
why doesn't this method return a list of recipients?

Reply:
if we add more properties to the recipient class later, we don't want to change this method.